fdbserver/DataDistribution.actor.cpp

```diff
@@ -2927,12 +2957,24 @@ ACTOR Future<Void> teamTracker(DDTeamCollection* self, Reference<TCTeamInfo> tea
 				if(lastPriority != team->getPriority()) {
 					self->priority_teams[lastPriority]--;
 					self->priority_teams[team->getPriority()]++;
+					if (lastPriority == SERVER_KNOBS->PRIORITY_TEAM_0_LEFT &&
+					    team->getPriority() < SERVER_KNOBS->PRIORITY_TEAM_0_LEFT) {
+						zeroServerLeftLogger = Void();
+					}
+					if (logTeamEvents) {
+						auto dataLoss = team->getPriority() == SERVER_KNOBS->PRIORITY_TEAM_0_LEFT;
+						auto severity = dataLoss ? SevError : SevInfo;
```

Comment:
We can get the "DataLoss" event when a cluster is down for a long time but later brought up. 
That might happen in simulation, which will cause simulation test fails, causing false positive failures.

I'd suggest changing it to
auto severity = dataLoss ? SevWarnAlways : SevInfo;